src/components/InlineCodeBlock/types.ts

```diff
@@ -0,0 +1,11 @@
+import {TextStyle, ViewStyle} from 'react-native';
+import type {TDefaultRenderer, TDefaultRendererProps, TText} from 'react-native-render-html';
+
+type InlineCodeBlockProps<TComponent extends TText> = {
+    TDefaultRenderer: TDefaultRenderer<TComponent>;
+    textStyle: TextStyle;
+    defaultRendererProps: TDefaultRendererProps<TComponent>;
+    boxModelStyle: ViewStyle & TextStyle;
```

Comment:
```suggestion
    boxModelStyle: StyleProp<ViewStyle>;
```

Based on my suggestion above.